content/articles/a-range-input-for-scientific-applications.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "A Range Input for Scientific Applications"
+date: 2018-08-23
+type: "post"
+author_name: "Nik Nyby"
+author_url: "https://ctl.columbia.edu/about/team/nyby/"
+lede: "This post describes how to change the behavior of an \u003Cinput type=range\u003E
```

Comment:
@zmustapha are these `<` and `>` character encoding changes necessary? hugo 0.35 seems to parse `<` and `>` in the lede correctly.